.cirrus.yml

```diff
@@ -285,17 +287,28 @@ success_task:
     - "centos9_build"
   gce_instance: *standard_gce_x86_64
   env:
-    CIRRUS_SHELL: "/bin/sh"
+    API_URL_BASE: "https://api.cirrus-ci.com/v1/artifact/build/${CIRRUS_BUILD_ID}"
+    EXP_BINS: >-
+        netavark
+        netavark.debug
+        netavark.info
+        netavark.aarch64-unknown-linux-gnu
+        netavark.debug.aarch64-unknown-linux-gnu
+        netavark.info.aarch64-unknown-linux-gnu
   clone_script: *noop
   bin_cache: *ro_bin_cache
   # The paths used for uploaded artifacts are relative here and in Cirrus
   script:
-    # Cross-compiled binary artifact is required by other CI systems
-    - curl --fail --location -O --url https://api.cirrus-ci.com/v1/artifact/build/${CIRRUS_BUILD_ID}/build_aarch64/armbinary.zip
+    - set -x
+    - curl --fail --location -O --url ${API_URL_BASE}/build_aarch64/armbinary.zip
     - unzip armbinary.zip
     - rm -f armbinary.zip
     - mv bin/* ./
     - rm -rf bin
+  artifacts_test_script:  # Other CI systems depend on all files being present
+    - ls -la
```

Comment:
Are we using output of this `ls -la` anywhere or is it only for debugging.

Reply:
Yes it shows the files that are actually present, since they are renamed in the build task. It makes it easier to diagnose any test failure, as opposed to requiring a developer to manually download the artifact zip to check.